public/v4/apps/fider.yml

```diff
@@ -0,0 +1,51 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        documentation: Taken from https://hub.docker.com/r/getfider/fider/
```

Comment:
There is not much documentation on dockerhub. Maybe link this one? https://getfider.com/docs/hosting-instance